### ISSUE TITLE
Allow instructors to update book configurations

### ISSUE THREAD

**Issue by awhicks** [OpenDSA/OpenDSA-LTI#176]
One version of this is adding the update_configuration button to books owned by an instructor in the admin interface
	lti/app/models/ability.rb 103 - add update_configuration
	make sure admin view adds update_configuration button that works to instructor accounts

A better solution is to add a button in the course configuration page (where you get dumped after creating a course offering) that takes you to a form to upload a new config file. This button (and the others) need good documentation and instructions for instructors.

**Comment by awhicks** [comment]
This is potentially being worked on by Katie in Fall 2024

**Comment by s-edwards** [comment]
Sorry to butt in, but there is already an "update configuration" link in the admin interface when showing the list of book instances, so I think this capability is already there in the admin interface (even though it is a link instead of a button). But instructors without admin access cannot use it, and it is missing from the course configuration page if you want to add it there.

**Comment by awhicks** [comment]
Yes, exactly! We just want to either: allow that button in the admin interface to instructors, or better put the functionality behind that link in a better place (next to the other buttons its use requires) so you don't have to jump back and forth to complete the workflow.
